app/assets/stylesheets/application.scss

```diff
@@ -1,6 +1,7 @@
 @import 'variables';
 @import 'spotlight/variables_bootstrap';
 @import 'bootstrap';
+@import 'bootstrap-overrides';
```

Comment:
Could these just go into the current variables file?

Reply:
No. Because of the import order. We could look at modifying the bootstrap import first, but this felt like the logical way forward.